serializers/src/OBJ/objSerializer.ts

```diff
@@ -67,7 +68,13 @@ export class OBJExport {
             }
 
             for (var i = 0; i < trunkVerts.length; i += 3) {
-                output.push("v " + trunkVerts[i] + " " + trunkVerts[i + 1] + " " + trunkVerts[i + 2]);
+                // Babylon default is left handed, while OBJ default is right handed
+                // By default we serialize in OBJ inverting the X vertices
+                if(sceneUsesRightHandedSystem) {
+                    output.push("v " + trunkVerts[i] + " " + trunkVerts[i + 1] + " " + trunkVerts[i + 2]);
+                } else {
+                    output.push("v " + -trunkVerts[i] + " " + trunkVerts[i + 1] + " " + trunkVerts[i + 2]);
```

Comment:
Got two questions here:

1) You are changing the default behavior (which just serialized all of the verts). Is that the right behavior for left-handed systems? Were we wrong all this time?
2) Usually we invert the 'z' axis. I might not know 100% how OBJ works - is the axis order different?

Reply:
1. Seems so, but I'm not the best on that topic ... maybe @Drigax can jump in here to validate, which suggested this in the first place

2. Also can't answer, I just know that this changes made my scenes exported correctly @Drigax could you help here ?  https://forum.babylonjs.com/t/obj-export-mirrored/10835/10

Reply:
I am totally fine with the changes if it is also a bug fix. will wait for @Drigax for a final approval

Reply:
I agree with @RaananW's suggestion, since Babylon's axes are setup as +Z Forward, +X right and +Y up, we should invert the Z axis. currently, we were inverting both Z and X with your proposed fix, imported into blender with default settings.